presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoMetadata.java

```diff
@@ -101,7 +105,7 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
         ImmutableList.Builder<SchemaTableName> tableNames = ImmutableList.builder();
         for (String schemaName : schemaNames) {
             for (String tableName : mongoSession.getAllTables(schemaName)) {
-                tableNames.add(new SchemaTableName(schemaName, tableName.toLowerCase(ENGLISH)));
+                tableNames.add(new SchemaTableName(schemaName, tableName));
```

Comment:
While the `toLowerCase` is redundant, I don't think this change is required for this PR. I would suggest separating a commit.